packages/zoe/test/unitTests/contracts/test-zoeHelpersWZcf.js

```diff
@@ -0,0 +1,129 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import '@agoric/install-ses';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import test from 'ava';
+
+import { E } from '@agoric/eventual-send';
+import bundleSource from '@agoric/bundle-source';
+
+// noinspection ES6PreferShortImport
+import { makeZoe } from '../../../src/zoeService/zoe';
+import { setup } from '../setupBasicMints';
+import fakeVatAdmin from './fakeVatAdmin';
+import { swap } from '../../../src/contractSupport';
+import { assertPayoutAmount } from '../../zoeTestHelpers';
+
+const contractRoot = `${__dirname}/zcfTesterContract`;
+
+test(`zoeHelper with zcf - swap`, async t => {
+  const {
+    moolaIssuer,
+    moola,
+    moolaMint,
+    simoleanIssuer,
+    simoleanMint,
+    simoleans,
+  } = setup();
+  const zoe = makeZoe(fakeVatAdmin);
```

Comment:
Would using this helper improve this? https://github.com/Agoric/agoric-sdk/blob/88d76f4ce59ffa733bf2f1209af4950a18505345/packages/zoe/test/unitTests/zcf/test-zcf.js#L17